docker/guardian/tls/docker-compose.yaml

```diff
@@ -0,0 +1,93 @@
+# - guardian-ui.fedimint.my-super-host.com
+# - gatewayd.fedimint.my-super-host.com
+# - gateway-ui.fedimint.my-super-host.com
+# - rtl.fedimint.my-super-host.com
+# - xmpp.fedimint.my-super-host.com
+
+version: "3.3"
+
+services:
+  ### START_TRAEFIK ###
+  traefik:
+    image: "traefik:v2.10"
+    container_name: "traefik"
+    command:
+      #- "--log.level=DEBUG"
+      - "--api.insecure=true"
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.websecure.address=:443"
+      - "--certificatesresolvers.myresolver.acme.tlschallenge=true"
+      - "--certificatesresolvers.myresolver.acme.storage=/letsencrypt/acme.json"
+    ports:
+      - "443:443"
+    volumes:
+      - "letsencrypt_data:/letsencrypt"
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+  ### END_TRAEFIK ###
+  
+  ### START_FEDIMINTD ###
+  fedimintd:
+    image: fedimint/fedimintd:v0.3.0
+    volumes:
+      - fedimintd_data:/data
+    ports:
+      - "0.0.0.0:8173:8173"
+    environment:
+      - FM_BITCOIN_RPC_KIND=bitcoind
+      - FM_BITCOIN_RPC_URL=http://bitcoin:bitcoin@bitcoind:38332
+      - FM_BITCOIN_NETWORK=signet
+      - FM_BIND_P2P=0.0.0.0:8173
+      - FM_P2P_URL=fedimint://fedimint.my-super-host.com:8173
+      - FM_BIND_API=0.0.0.0:80
+      - FM_API_URL=wss://fedimintd.fedimint.my-super-host.com
+      - FM_EXTRA_DKG_META=chat_server_domain=xmpp.fedimint.my-super-host.com
+    restart: always
+    platform: linux/amd64
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.fedimintd.loadbalancer.server.port=80"
+      - "traefik.http.routers.fedimintd.rule=Host(`fedimintd.fedimint.my-super-host.com`)"
+      - "traefik.http.routers.fedimintd.entrypoints=websecure"
+      - "traefik.http.routers.fedimintd.tls.certresolver=myresolver"
+  ### END_FEDIMINTD ###
+
+  ### START_GUARDIAN_UI ###
+  guardian-ui:
+    image: fedimintui/guardian-ui:0.3.0
+    environment:
+      - PORT=80
+      - REACT_APP_FM_CONFIG_API=wss://fedimintd.fedimint.my-super-host.com
+    depends_on:
+      - fedimintd
+    restart: always
+    platform: linux/amd64
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.guardian-ui.loadbalancer.server.port=80"
+      - "traefik.http.routers.guardian-ui.rule=Host(`guardian-ui.fedimint.my-super-host.com`)"
+      - "traefik.http.routers.guardian-ui.entrypoints=websecure"
+      - "traefik.http.routers.guardian-ui.tls.certresolver=myresolver"
+  ### END_GUARDIAN_UI ###
+
+  ### START_BITCOIND ###
+  bitcoind:
+    image: fedimint/mutinynet-bitcoind:master
+    command: --rpcuser=bitcoin --rpcpassword=bitcoin -zmqpubrawblock=tcp://[::]:48332 -zmqpubrawtx=tcp://[::]:48333
+    ports:
+      - 0.0.0.0:38333:38333
+    volumes:
+      - "bitcoin_datadir:/root/.bitcoin"
+    restart: always
+    platform: linux/amd64
+  ### END_BITCOIND ###
+
+volumes:
+  letsencrypt_data:
+  fedimintd_data:
+  bitcoin_datadir:
```

Comment:
One thing I removed from my scripts which you may want to put back was xmpp. I'm unsure if that was intentionally left out of here or not too.

Reply:
nope we're pulling all xmpp out from these

Reply:
Is chat going to be removed completely in fedimint?

Reply:
removing here in the open source repo, fedi specifically uses it not a general fedimint thing

Reply:
I know other mints that still want to operate and communicate through XMPP through the fedi app. 

Removing it here, and keeping XMPP in fedi, is prioritizing secret functionality only known by fedi deployments.